distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -833,23 +849,112 @@ public void testCswQueryByTitle() throws Exception {
 
     @Test
     public void testCswQueryForMetacardXml() throws Exception {
-        String titleQuery = getCswQuery("title",
-                "myTitle",
-                "application/xml",
-                "urn:catalog:metacard");
 
-        given().contentType(ContentType.XML)
-                .body(titleQuery)
-                .when()
-                .post(CSW_PATH.getUrl())
-                .then()
-                .assertThat()
-                .body(hasXPath("/GetRecordsResponse/SearchResults/metacard/@id",
-                        is(metacardIds[GEOJSON_RECORD_INDEX])),
-                        hasXPath("/GetRecordsResponse/SearchResults/@numberOfRecordsReturned",
-                                is("1")),
-                        hasXPath("/GetRecordsResponse/SearchResults/@recordSchema",
-                                is("urn:catalog:metacard")));
+        // The first few bytes of the metacard's base-64 encoded thumbnail image
+        String thumbNailBase64EncSubstring = "/9j/4AAQSkZJRgABAQAAAQABAAD";
+        String metacardTitle = "myTitle";
+        String extractXmlRegex = ".*?(<metacard.*?</metacard>).*";
+
+        // Define the assertions that test the query results conform to expectations
+        List<Matcher<?>> assertions = getXpathMatchers(METACARD_URI, 1);
+        assertions.add(hasXPath("//*[@name='thumbnail']/*/text()",
+                startsWith(thumbNailBase64EncSubstring)));
+        assertions.add(hasXPath("//*[@name='title']/*/text()", equalTo(metacardTitle)));
+        assertions.add(hasXPath("//pos/text()", equalTo("30.0 10.0")));
+
+        // Query for CSW result and assert expected results
+        ValidatableResponse response = getAndValidateCswResponse(metacardTitle,
+                METACARD_URI,
+                assertions);
+
+        // Extract the port of the response that can be transformed into a metacard
+        String metacardXml = extractRecord(response, extractXmlRegex);
+
+        // Get the metacard XML input transformer and attempt to create a metacard object
+        Metacard metacard = getInputTransformer("(id=xml)").transform(IOUtils.toInputStream(
```

Comment:
This doesn't go through the same flow as a query over csw. I think we should be creating the metacards by doing a loopback query specifying this as the outputformat, so that we verify things like caching, query plugins, and schema validation in cxf play nicely with each schema. You could also keep this step and do the above as an additional, final test for extra granularity, I guess.